packages/backend/src/services/index.ts

```diff
@@ -1,7 +1,9 @@
 import { Application } from '../declarations';
 import users from './users/users.service';
+import nfcReaders from './nfc-readers/nfcReader.service';
```

Comment:
I would rename the service and model files to use lower-case-hyphenated names.

Reply:
resolved by: 7f6fca830e15635acf95b520a301f99af5847662